Makefile

```diff
@@ -1,9 +1,10 @@
 ORG=integreatly
 NAMESPACE=integreatly
+RHMDSNAMESPACE=mobile-developer-console
```

Comment:
What is this for?

Reply:
its for mdc https://github.com/integr8ly/installation/blob/0dccb5958f2b746532b767a2d4da2d5e3ca054a7/roles/rhsso-user/tasks/main.yml#L22 doens't actually do anything at the moment

Reply:
If its not part of user rhsso lets leave it out

Reply:
has a chat with John, removing mdc stuff

Reply:
:+1: